### ISSUE TITLE
Romanian bundle loading successfully, but strings don't show up

### ISSUE THREAD

**Issue by YellOw139** [Anuken/Mindustry#2524]
**Platform**: All

**Build**: 104.10 but I assume it happens in BE too (didn't know how to test it since I cannot find the exe if there's one)

**Issue**:
Romanian bundle loading successfully, but strings don't show up (the string name is shown instead). 

Could be related to the Romanian characters ș, ț, ă, â, î.

**Steps to reproduce**: Saved the Romanian bundle file as bundle.properties in the Mindustry directory. Encoding was set to Unicode when saving. The file needs to be saved that way, otherwise the characters ș, ț, ă, â, î are not saved.

**Link(s) to mod(s) used**: None

**Save file**: None. Started the game with clean data and jumped right into the tutorial.

**Crash report**: None.

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by YellOw139** [comment]
It's interesting to note that I can also opt to not use the special characters (using a,i,s,t instead) and save the file in ANSI. However, the resulting translation is inaccurate and not suitable for an official game translation.

**Comment by Anuken** [comment]
Save the bundle as UTF-8.

> ș, ț, ă, â, î

These are (probably) not supported by the font, and can't really be added as I didn't *make* the font, and adding in characters from other fonts would look terrible. If you want, you can try to find a font that has these characters in a similar font, but I doubt the result will look acceptable.

**Comment by YellOw139** [comment]
I found some fonts that have the required glyphs.

I think [Axion RND](https://www.myfonts.com/fonts/typeinnovations/axion-rnd?tab=individualStyles) fits your game the best of all I tried. Since it looks so good I decided to mention it, but sadly it doesn't seem to offer an app license.

I also found [Neuropol X](https://www.myfonts.com/fonts/typodermic/neuropol-x). It comes in a variety of styles and offers an app license. It looks pretty decent.

In terms of free fonts, I think [Asimov Print A or C](https://www.fontzillion.com/fonts/cannot-into-space-fonts/asimov-print?utm_source=fontsquirrel.com&utm_medium=matcherator_link&utm_campaign=asimovprint) are about as close as we can get. They are free and are licensed under Apache 2.0.

Your game, your choice. It seems that ț, ş and ă (in this exact order) are the really rare glyphs. For now we may be able to use an inaccurate translation (I can replace glyphs until it works properly in-game). I will test what works and provide an inaccurate translation for temporary use if you're interested. People will understand anyway; we often have this glyph issue here.
